reconciler/retry_test.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2020 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconciler
+
+import (
+	"errors"
+	"testing"
+
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+
+	v1 "k8s.io/api/autoscaling/v1"
```

Comment:
nit: the package name is already v1 so the identifier is not necessary here.

```suggestion
	"k8s.io/api/autoscaling/v1"
```

Reply:
goimports adds these, leave it. (thank gomod 🙄 )